ci-operator/step-registry/ipi/install/powervc/install/ipi-install-powervc-install-commands.sh

```
@@ -1,307 +1,867 @@
+#   $1 - Line number where error occurred
+#   $2 - Exit code of failed command
+# Output:
+#   Error message to stderr
+# Side Effects:
+#   Calls prepare_next_steps for cleanup
+function error_handler() {
+	local line_no=$1
+	local exit_code=$2
+	log_error "Script failed at line ${line_no} with exit code ${exit_code}"
+	prepare_next_steps
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)